src/Administration/Resources/app/administration/src/module/sw-order/page/sw-order-detail/index.js

```diff
@@ -108,7 +108,9 @@ export default {
         },
 
         orderRepository() {
-            return this.repositoryFactory.create('order');
+            return this.repositoryFactory.create('order', null, {
+                useSync: true,
+            });
```

Comment:
I'm not sure why you have to use the sync api at all. The component was working before, therefore we have a regression in our `sw-entity-tag-select`/`sw-entity-multi-select` components. Please have a look there as it could fix potential other issues too :smile: